index.ts

```diff
@@ -65,16 +65,16 @@ Object.keys(servicesByVersion).forEach(serviceName => {
     value: function(options) {
       options = options || {};
 
-      // previously, AlchemyAPI did not require a version to be specified
+      // previously, AlchemyAPI did not require a serviceVersion to be specified
       if (serviceName.indexOf('alchemy_') === 0) {
-        options.version = 'v1';
+        options.serviceVersion = 'v1';
       }
 
-      const Service = servicesByVersion[serviceName][options.version];
+      const Service = servicesByVersion[serviceName][options.serviceVersion];
```

Comment:
This is a breaking change 🔥 

Users should still be able to use

```js
var conversationV1 = watson.conversation({
  version: 'v1',
  version_date: '2017-05-26'
});
```

Reply:
Okay ... I see now.  We documented this style in past releases.  It's good to see that we have removed the doc on this.  I will think about how we can handle that.